css/registrationForm.html

```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
+<html lang="bg">
+<head>
+    <meta content="text/html; charset=UTF-8" http-equiv="Content-Type" />
+    <link rel="stylesheet" type="text/css" href="registrationForm.css">
+    <title>Registration form</title>
+</head>
+<body>
+    <form method="POST">
+        <h5>РЕГИСТРАЦИЯ</h5>
+        <input type="text" class="userField" placeholder="E-mail">
+        <br>
+        <input type="text" class="userField" placeholder="Име">
+        <br>
+        <input type="text" class="userField" placeholder="Фамилия">
+        <br>
+        <input type="password"class="userField" placeholder="Парола">
+        <br>
+        <input type="password" class="userField" placeholder="Повторна парола">
+        <br>
+        <input type="submit" class="userField" id="registrationButton" value="Регистрация">
+    </form>
+</body>
+</html>
```

Comment:
I meant adding the snapshot as a comment in the PR, please remove it from commit and add as comment. I will also fix this on the slides. Thanks!